src/components/EventListCard/Modal/Preview/EventListCardPreviewModal.tsx

```
@@ -437,16 +441,50 @@ const PreviewModal: React.FC<InterfacePreviewEventModalProps> = ({
-              <label htmlFor="ispublic">{t('isPublic')}?</label>
-              <Form.Switch
-                id="ispublic"
-                type="checkbox"
-                data-testid="updateIsPublic"
-                className={`me-4 ${styles.switch}`}
-                checked={publicchecked}
-                onChange={() => setPublicChecked(!publicchecked)}
-                disabled={!canEditEvent}
-              />
+              <div className="mb-3">
+                <label className="mb-2">{t('visibility')}</label>
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)